module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,48 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <vector>
```

Comment:
You have included array,cstdint and vector in the header,so you do not need to include them here.